src/client.ts

```diff
@@ -206,6 +220,10 @@ class NoopClient implements Client {
 		return '';
 	}
 
+	sessionId() {
+		return 0;
```

Comment:
How about returning `null` instead? I think it's more accurate to say that there is no session id, rather than returning an arbitrary number

Reply:
Sure `null` works as well.

Reply:
Does it matter though?
Note that this is part of the `NoopClient` implementation, which is some kind of an empty mock, not even sure where it's used (maybe for prototyping?).

I've changed it to `-1` as per documentation. `null` would voilate the expected `number` type - I assume `deviceId()` returns as empty string instead of `null` for similar reasons.